crypto/params_idx.c.in

```diff
@@ -0,0 +1,21 @@
+/*
+ * {- join("\n * ", @autowarntext) -}
+ *
+ * Copyright 2023 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+{-
+use OpenSSL::paramnames qw(produce_decoder);
```

Comment:
@paulidale what about moving this use statement to util/dofile.pl? Maybe it could help.

Reply:
Or actually IMO this should not be needed at all? But you just need to fully specify the `paramnames::produce_decoder` below.

Reply:
copy/paste programming :)  I don't know what I'm doing.